apps/base-docs/tutorials/sidebars.js

```diff
@@ -182,6 +182,17 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: 'category',
```

Comment:
Generally we would like to have more than one item in a category before creating a new category, so let's add this elsewhere and look to perhaps create a new one in the future